src/js/user-profile/reducers/profile.js

```diff
@@ -15,14 +16,21 @@ const initialState = {
   gender: null,
   accountType: null,
   savedForms: [],
-  prefillsAvailable: []
+  prefillsAvailable: [],
+  loading: true
```

Comment:
So anytime the app is first loaded this will be defaulted to `true`, and only changed to `false` once we finish the fetch of the data? Makes sense to me.